### PR TITLE
[add] Contributing guidelines in readme

### PR DESCRIPTION
**Pull request by malikakarsh**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,45 @@
...
+# :arrow_down:Important
+
+  
+Follow these steps:hammer: for contributing:
+
+:green_heart:Fork the repo.
+
+:pushpin:Create a separate branch for any bug/feature with branch name: <grid/bug>
+
+:white_check_mark:Commit once a program is complete.
```
Comment (nishihere19) [inline README.md]: Can you change it to 'Commit once a feature is done/a bug is fixed'.

File: README.md [hunk 2]
```diff
@@ -1,2 +1,45 @@
...
+
+  
+
+# :bookmark:Code advice
+
+Kindly maintain a clean code with proper comments.
+
+Commit messages should be written carefully and describe the changes made.
+
+Create separate PR for a single program update/bug fix.
```
Comment (nishihere19) [inline README.md]: Change program update to 'feature'.

Comment (nishihere19) [reply README.md]: Also add "Happy contributing" in the end.

**Review by nishihere19 — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (malikakarsh) [reply README.md]: No

File: README.md (hunk 2 above)
Comment (malikakarsh) [reply README.md]: okay

**Review by nishihere19 — APPROVED** [review]

**Pull request merged**